### PR TITLE
Transport: convert name to string

### PR DESCRIPTION
**Pull request by savka2017**

### CODE REVIEW

File: app/models/transport.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 class Transport < ApplicationRecord
-  TRANSPORT_NAMES = %i[car train bus no_transport unknown].freeze
-
-  enum name: TRANSPORT_NAMES
-
-  validates_presence_of :name
+  enum name: {
+    car: 'Can',
```
Comment (ChernikovVolodymyr) [inline app/models/transport.rb]: May be ```Car``` need here?

Comment (savka2017) [reply app/models/transport.rb]: Sure...

Comment (savka2017) [reply app/models/transport.rb]: https://github.com/vova/war_shelter/pull/26/commits/4fba1553e3a87f5a1322fa8ee4330cbd9641736f

Comment (savka2017) [reply app/models/transport.rb]: https://github.com/vova/war_shelter/pull/26/commits/4fba1553e3a87f5a1322fa8ee4330cbd9641736f

**Review by ChernikovVolodymyr — APPROVED** [review]

**Review by yuramon — APPROVED** [review]

**Review by oleksandr-kychun — APPROVED** [review]

**Pull request merged**